Fetch providers

Ton033 opened the pull request:
+ added all providers

Ton033 committed:
fix minor readme and delete old readme

Ton033 committed:
added logo and tab title

+ added all providers

Kenzirey commented on frontend/index.html:
```diff
@@ -2,9 +2,9 @@
+    <link rel="icon" type="image/png" href="public/images/logo.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + Vue</title>
+    <title>Learniverse Connect</title>
```
Nice find! I completely missed this. Cookie from me

Ton033 committed:
added windows scroll to

Kenzirey committed:
Fixed FavoritePage

Made it mobile friendly, fixed button placement. Removed description, added difficulty level instead.

Kenzirey committed:
Merge branch 'fetch-providers' of https://github.com/Kenzirey/Web-App-Group-15 into fetch-providers
